### ISSUE TITLE
The code inside "Disable/enable all linters" is actually parsed

### ISSUE THREAD

**Issue by malyw** [sasstools/sass-lint#944]
**Sass Lint version:** 1.10.1
**Configuration:** Default
**Expected behavior:**
Sass-lint to ignore all the errors inside
// sass-lint:disable-all
...
// sass-lint:enable-all
blocks, as described at https://github.com/sasstools/sass-lint/blob/develop/docs/toggle-rules-in-src.md

**Actual behavior:**
For the popular library include-media https://github.com/eduardoboucas/include-media/blob/master/dist/_include-media.scss
Sass-linter cannot parse the block (which is the valid Sass code):
```scsss
@include media(slice($conditions, 2)...) {
        @content;
      }
```
ans says `error  Please check validity of the block starting from line #X  Fatal`

The same behavior if the block is wrapped in the "disable-all" rule:
```scss
// sass-lint:disable-all
      @include media(slice($conditions, 2)...) {
        @content;
      }
      // sass-lint:enable-all
```

**The problem**
Despite just Sass-lint to actually check the ignored code, looks like it has problems with functions followed by three dots (variable arguments):
http://sass-lang.com/documentation/file.SASS_REFERENCE.html#variable_arguments
https://www.sitepoint.com/sass-multiple-arguments-lists-or-arglist/

**Comment by DanPurdy** [comment]
I'm sorry but this isn't how these comments are supposed to work. 

Our AST/parser still needs to be able to parse absolutely everything, the comments are only there to prevent warnings and errors generated by the rules, if there's a parse error as you're showing here you need to report it to [Gonzales-pe](https://github.com/tonyganch/gonzales-pe).

The AST even has to read and tokenize our disable comments for us to be able to act on them too so you can see why this wouldn't ever be something that we would change.

**Comment by bgriffith** [comment]
Hey @malyw - Just a heads up that this has already been fixed in the parser https://github.com/tonyganch/gonzales-pe/issues/219 and will be fixed in sass-lint once the parser does a new release :+1:
